compiler-core/src/erlang/tests/panic.rs

```diff
@@ -22,15 +22,29 @@ pub fn main() {
     );
 }
 
-// https://github.com/gleam-lang/gleam/issues/2176
 #[test]
-fn piped() {
```

Comment:
Please put back this function and comment that have been deleted.

Reply:
Got it. Sorry, thought that the comment was referring to a issue that was closed.

Reply:
Yes, it's a test to prevent a regression on that bug.